template/App.tsx

```diff
@@ -66,7 +66,10 @@ const App = () => {
 
   return (
     <SafeAreaView style={backgroundStyle}>
-      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
+      <StatusBar
+        barStyle={isDarkMode ? "light-content" : "dark-content"}
```

Comment:
Oh I missed that. But nice that we have a linting pipeline that caught the double quotes.